### PR TITLE
6308 Quick, crude stub of About page

### PR DESCRIPTION
**Pull request by godfreyyeung**
Fixes AB#6308

### CODE REVIEW

File: src/pages/about.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { Flex, Box, Center } from "@chakra-ui/react";
+
+const AboutPage = () => {
+  return (
+    <Flex direction="column" width="100%">
+      <Flex direction="row" bg="teal" height="300px">
+        <Box flex="1"></Box>
+        <Box flex="5">
+          <h1>About</h1>
```
Comment (TylerMatteo) [inline src/pages/about.tsx:9]: Not worried about this for a stub, just making a note we can get into the habit of using [Chakra's heading component](https://chakra-ui.com/docs/typography/heading) for these.

File: src/pages/about.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { Flex, Box, Center } from "@chakra-ui/react";
+
+const AboutPage = () => {
+  return (
+    <Flex direction="column" width="100%">
+      <Flex direction="row" bg="teal" height="300px">
+        <Box flex="1"></Box>
```
Comment (TylerMatteo) [inline src/pages/about.tsx:7]: I think "spacer divs" like these are an anti-pattern. What we probably want here is a centered container with a max-width.

File: src/pages/about.tsx (hunk 1 above)
Comment (godfreyyeung) [reply src/pages/about.tsx:9]: Okay will try that out

File: src/pages/about.tsx (hunk 2 above)
Comment (godfreyyeung) [reply src/pages/about.tsx:7]: Yeah, just stubbing for now so made it really quick.

**Review by TylerMatteo — APPROVED** [review]

**Pull request merged**